make/modules/java.management/Lib.gmk

```diff
@@ -27,19 +27,9 @@
 
 include LibCommon.gmk
 
-################################################################################
-## Build libmanagement
-################################################################################
-
```

Comment:
Why remove the comment header. This pattern is used throughout the `make/modules/*/Lib.gmk` files.

Reply:
My motivation was that the comment brings not much value because it just states the obvious. But if you like I can bring back the comment.

Reply:
I brought back the comment, maybe it is better to keep it because of consistency.

Reply:
Thank you. I have introduced these headers everywhere to to create consistency and an easy way to search for where native libraries are compiled, so they are definitely useful even if that is not obvious from a single example.